src/components/application_manager/CMakeLists.txt

```diff
@@ -389,6 +394,11 @@ set(PATHS
 )
 collect_sources(SOURCES "${PATHS}" "${EXCLUDE_PATHS}")
 
+if (NOT REMOTE_CONTROL)
+    list(REMOVE_ITEM SOURCES
+        ${COMPONENTS_DIR}/application_manager/src/core_service.cc)
```

Comment:
@JackLivio core service provide  proxy to application manager staff needed for remote controll